cypress/integration/main_page_spec.js

```diff
@@ -91,6 +91,31 @@ describe('Dashboard Home Page', () => {
       });
     });
 
+    xit('displays a date picker in metrics section', () => {
```

Comment:
Does this not run because the xit? 
And you turned it off because it isn't currently on the home page?

Reply:
yes.  the test won't run with `xit`